src/runtime-rs/crates/runtimes/common/src/types/trans_into_shim.rs

```diff
@@ -6,37 +6,16 @@
 
 use std::{
     any::type_name,
-    convert::{Into, TryFrom, TryInto},
-    time,
+    convert::{Into, TryFrom},
 };
 
 use anyhow::{anyhow, Result};
 use containerd_shim_protos::api;
 
 use super::{ProcessExitStatus, ProcessStateInfo, ProcessStatus, Response};
+use super::utils::option_system_time_into;
 use crate::error::Error;
 
-fn system_time_into(time: time::SystemTime) -> ::protobuf::well_known_types::timestamp::Timestamp {
```

Comment:
Because if I didn't move these two functions to utils, I would have to make `trans_into_shim` module public and use this module outside of `common` module. I think it's not graceful.